src/components/Avatar/Avatar.tsx

```diff
@@ -90,6 +192,7 @@ export const Avatar: FC<AvatarProps> = React.forwardRef(
       style = {},
       fontSize = '18px',
       iconProps,
+      statusIcons = {},
```

Comment:
rename to `statusIndicators` or `statusItems` as they may be icons, divs, buttons et cetera.

Reply:
Same for the other types too.